state/application.go

```diff
@@ -1949,64 +1949,46 @@ func (a *Application) SetDownloadedIDAndHash(id, hash string) error {
 	return nil
 }
 
-func checkSeriesForSetCharm(currentPlatform *Platform, charm *Charm, ForceBase bool) error {
+func checkSeriesForSetCharm(currentPlatform *Platform, ch *Charm, ForceBase bool) error {
```

Comment:
Does this method duplicate validation also done in a core/charm method? It looks very similar if not the same.  Is it the BaseSelector?

If we still need this one, rather the equivalent method from core, s/checkSeriesForSetCharm/checkBaseForSetCharm/